tasks/consensus/task.go

```diff
@@ -0,0 +1,57 @@
+package consensus
+
+import (
+	"context"
+
+	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/sentinel-visor/model"
+	"github.com/filecoin-project/sentinel-visor/model/consensus"
+	visormodel "github.com/filecoin-project/sentinel-visor/model/visor"
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/label"
+)
+
+type Task struct {
+}
+
+func NewTask() *Task {
+	return &Task{}
+}
+
+func (t *Task) ProcessTipSets(ctx context.Context, child, parent *types.TipSet) (model.Persistable, *visormodel.ProcessingReport, error) {
```

Comment:
Check that child.Parents() == parents

Reply:
That's already handled by the indexer before calling this.